main.go

```diff
@@ -242,6 +234,21 @@ func getExportData(fs *fullstory.Client, bundleID int) (fullstory.ExportData, er
 	return nil, errors.Wrap(fsErr, fmt.Sprintf("Unable to fetch export data. Tried %d times.", maxAttempts))
 }
 
+func getRetryInfo(err error) (bool, time.Duration) {
+	if statusError, ok := err.(fullstory.StatusError); ok {
+		// If the status code is NOT 429 and the code is below 500 we will not attempt to retry
+		if statusError.StatusCode != http.StatusTooManyRequests && statusError.StatusCode < 500 {
+			return false, defaultRetryAfterDuration
```

Comment:
wondering if the retry after we return here should be a stupid long duration or something so people don't by mistake hose fullstory servers if there are client errors?

Reply:
I guess define "stupid long"? Right now the default wait is 10s, and it doesn't seem like a call every 10s is often enough to hose anything. Keep in mind that without this retry mechanism, the error would bubble up to the main function, which would then wait 30s and try again anyway. So we're not drastically changing the overall retry rate when the error is something "catastrophic" and can't be resolved by waiting.

Reply:
I guess I said stupid long because I couldn't define it well :p 

agree with your thoughts!